react/Price.js

```diff
@@ -32,33 +32,32 @@ class Price extends Component {
     const formattedInstallmentPrice = formatNumber(installmentPrice, currencyOptions)
 
     return (
-      <div className="tc b fabriga">
+      <div className="vtex-product-summary__price tc fabriga">
         {showListPrice && (
-          <div className="pv1">
+          <div className="pv1 f6 normal">
             {showLabels &&
-              <div className="dib">
+              <div className="vtex-product-summary__price-list-price-label dib">
                 <FormattedMessage id="pricing.from" />
               </div>
             }
-            <div className="dib strike ph2">
+            <div className="vtex-product-summary__price-list-price dib strike ph2">
```

Comment:
`vtex-price`
`vtex-price-list__label`

Seems a little weird, don't you think? `vtex-price-list__label-price`